besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogHelper.java

```diff
@@ -285,11 +279,10 @@ private static void saveTrieLogsInFile(
   }
 
   @SuppressWarnings("unchecked")
-  private static IdentityHashMap<byte[], byte[]> readTrieLogsFromFile(
-      final String batchFileNameBase, final long batchNumber) {
+  static IdentityHashMap<byte[], byte[]> readTrieLogsFromFile(final String batchFileName) {
```

Comment:
Non blocking feedback - 

I am all for code reuse, but if we are going to allow for arbitrary import and export, the import files should be more readable and "createable".  


The ObjectOutputStream seems fine for backup/recovery of a pruning process, but when part of a general import/export process this file format is too inscrutable IMO.

At least for import/export we should serialize/deserialize these as json maps.  Key as the hash string, and the trielog itself as hex (or as a rich json object if we wanted to be super transparent).  In addition to being a bit more introspectable, it would allow us to create and import our own handcrafted trielogs when debugging